Source/Core/AudioCommon/Mixer.cpp

```diff
@@ -202,12 +210,24 @@ unsigned int Mixer::Mix(short* samples, unsigned int num_samples)
   }
   else
   {
-    m_dma_mixer.Mix(samples, num_samples, true, emulation_speed, timing_variance);
-    m_streaming_mixer.Mix(samples, num_samples, true, emulation_speed, timing_variance);
-    m_wiimote_speaker_mixer.Mix(samples, num_samples, true, emulation_speed, timing_variance);
-    m_skylander_portal_mixer.Mix(samples, num_samples, true, emulation_speed, timing_variance);
+    const bool is_ffw = Core::GetIsThrottlerTempDisabled();
```

Comment:
If I remember correctly, variables shouldn't use acronyms. I might be wrong but I think it would be better. `ffw` is not clear.

Reply:
Also, directly reading `Core::GetIsThrottlerTempDisabled()` here is likely fine, but maybe it could be passed in in a more thread friendly way? No big deal anyway.

Reply:
I actually determined when to reset audio on the main thread (though not in CoreTiming which actually handles fast-forwarding), in an earlier version of this code (https://github.com/nyanpasu64/dolphin/commit/61e7b78b7a01f8cad6516a9244020c32c960ac7d).

> The existing code updates `s_is_throttler_temp_disabled` on HotkeyScheduler but reads it (with `GetIsThrottlerTempDisabled()`) on the CPU and Video threads.

I replaced `s_is_throttler_temp_disabled` with an atomic to make it safe to access on multiple threads (which is already happening, with or without my PR merged). But perhaps it's still more reliable for the main thread to set a flag (read by the mixer thread) when the main thread starts and stops fast-forwarding.

- [ ] You'll most reliably detect when fast-forwarding starts and stops by making `CoreTimingManager` inform the mixer when `CoreTimingManager` itself stops fast-forwarding. But this is a nasty entanglement of concerns (`CoreTimingManager` shouldn't care about audio mixer resyncing?). Should I implement this change or not?
- How does other code read `s_is_throttler_temp_disabled`? Right now, `VideoConfig.cpp#IsVSyncActive()` calls `GetIsThrottlerTempDisabled()` rather than checking whether `CoreTimingManager` has started or stopped fast-forwarding. Should we disable vsync for a frame if the user starts and stops fast-forwarding within one frame, causing a sub-frame shift in emulation phase relative to vsync? This is somewhat theoretical and unrelated to my code changes. I still hope that something like #12035 can get merged.